CircleandCross2/Program.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+
+namespace CircleandCross2
+{
+	class Program
+	{
+
+		static void Main(string[] args)
+		{
+			try
+			{
+
+
+				Console.WriteLine("Tic Tac Toe");
+
+				string[,] array = new string[3, 3] { { "[ ]", "[ ]", "[ ]" }, { "[ ]", "[ ]", "[ ]" }, { "[ ]", "[ ]", "[ ]" } };
+
+				for (int i = 0; i < array.GetLength(0); i++)
+				{
+					for (int j = 0; j < array.GetLength(1); j++)
+					{
+						string arr = array[i, j];
+						Console.Write(arr);
+					}
+					Console.WriteLine();
+				}
+
+
+				for (int a = 1; a <= 5; a++)
+				{
+
+					Console.WriteLine("podaj pozycje 'o' w lini 'i'");
+					string tekst = Console.ReadLine();
+					int pozycja1i = int.Parse(tekst);
```

Comment:
Use `int.TryParse()` for parsing user input. If the input can't be correctly parsed program should ask again for the position.

Reply:
i changed it but not all works like i want

Reply:
You need to create a function that will handle user input. Inside it you have to use while or do while loop. Inside the loop, you need to check if passed input is a number, is in gameboard range, and if the position is taken. And only if all conditions are fulfilled you can put anything to the game board.

Reply:
I have this but i dont use function

Reply:
i do it